tsinfer/cli.py

```diff
@@ -177,13 +178,19 @@ def run_generate_ancestors(args):
     setup_logging(args)
     ancestors_path = get_ancestors_path(args.ancestors, args.samples)
     sample_data = tsinfer.SampleData.load(args.samples)
-    tsinfer.generate_ancestors(
+    ancestors = tsinfer.generate_ancestors(
         sample_data,
         progress_monitor=args.progress,
-        path=ancestors_path,
         num_flush_threads=args.num_flush_threads,
         num_threads=args.num_threads,
+        record_provenance=False,
     )
+    new_ancestors = ancestors.copy(path=ancestors_path)
```

Comment:
This is a big cost just for provenance, and will lead to a huge pause after the progress bar is done. I don't think it's worth it.

Maybe just skip this one and open an issue to track? There's no useful options in generate ancestors to record anywy

Reply:
Good point: I hadn't thought about the perf issues of making a copy. I assume there's no way to add provenance to the existing ancestors file without copying?

As you say, not worth spending time over, if we are re-doing the pipeline anyway, so we can just open an issue to track.